pkg/server/server.go

```diff
@@ -270,7 +281,12 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 			// the address, then we can send the Dial_REQ to the
 			// same agent. That way we save the agent from creating
 			// a new connection to the address.
-			backend, err = s.BackendManager.Backend(context.TODO())
+			ctx := context.Background()
+			if s.proxyStrategy == "destHost" {
+				addr := util.RemovePortFromHost(pkt.GetDialRequest().Address)
```

Comment:
Does this mean it's only possible to send requests to nodes? Pod IPs within the cluster are off limits?

Reply:
Will KAS send requests to POD IPs directly? I thought all requests will be just from KAS to Kubelet.